src/patterns/ProgressWizard/hooks/useHandleActions.ts

```diff
@@ -0,0 +1,116 @@
+import { useCallback } from 'react';
+import type { LoadingState, CallbackProps, FormStep } from '../types';
+import type { UseFormReturn, FieldValues } from 'react-hook-form';
+
+export const useHandleActions = ({
```

Comment:
more comments please on the overall hook and the individual hook arguments and return values

Reply:
`useProgressWizardActions`?

Reply:
I usually like to use "handle*" for internal function names, but not props or external naming